targets/TARGET_STM/TARGET_STM32F4/TARGET_STM32F437xG/TARGET_UBLOX_C030/system_stm32f4xx.c

```diff
@@ -215,10 +214,16 @@ void SystemInit(void)
 #endif /* DATA_IN_ExtSRAM || DATA_IN_ExtSDRAM */
 
   /* Configure the Vector Table location add offset address ------------------*/
+  SCB->VTOR = FLASH_BASE; /* Vector Table Relocation in Internal FLASH */
+
 #ifdef VECT_TAB_SRAM
-  SCB->VTOR = SRAM_BASE | VECT_TAB_OFFSET; /* Vector Table Relocation in Internal SRAM */
-#else
-  SCB->VTOR = FLASH_BASE | VECT_TAB_OFFSET; /* Vector Table Relocation in Internal FLASH */
+  int *vectors = (int *)SCB->VTOR;
```

Comment:
Well, I suppose it would mean that, but an extra copy step is worth it to have my platform boot and I can't see anything wrong with the platform initialising its interrupt vector table in `SystemInit()`.  If there is a proper/tested answer I'm happy for this PR to be dropped; I'm happy to wait a few days for that answer to come about.  If anyone wants working code for the `UBLOX_C030` platform I will point them to my fix branch in the meantime.